web/wp-content/themes/dctx-v2/inc/scripts.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * Custom scripts and styles.
+ *
+ * @package dctx
+ */
+
+/**
+ * Register Google font.
+ *
+ * @link http://themeshaper.com/2014/08/13/how-to-add-google-fonts-to-wordpress-themes/
+ *
+ * @author WDS
+ * @return string
+ */
+function dctx_font_url() {
+
+	$fonts_url = '';
+
+	/**
+	 * Translators: If there are characters in your language that are not
+	 * supported by the following, translate this to 'off'. Do not translate
+	 * into your own language.
+	 */
+	$roboto    = esc_html_x( 'on', 'Roboto font: on or off', 'dctx' );
+	$work_sans = esc_html_x( 'on', 'Work Sans font: on or off', 'dctx' );
+
+	if ( 'off' !== $roboto || 'off' !== $work_sans ) {
+		$font_families = array();
+
+		if ( 'off' !== $roboto ) {
+			$font_families[] = 'Roboto:400,700';
+		}
+
+		if ( 'off' !== $work_sans ) {
+			$font_families[] = 'Work Sans';
+		}
+
+		$query_args = array(
+			'family' => rawurlencode( implode( '|', $font_families ) ),
+		);
+
+		$fonts_url = add_query_arg( $query_args, '//fonts.googleapis.com/css' );
+	}
+
+	return $fonts_url;
+}
+
+/**
+ * Enqueue scripts and styles.
+ *
+ * @author WDS
+ */
+function dctx_scripts() {
+	/**
+	 * If WP is in script debug, or we pass ?script_debug in a URL - set debug to true.
+	 */
+	$debug = ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) || ( isset( $_GET['script_debug'] ) ) ? true : false; // WPCS: CSRF OK.
```

Comment:
:warning: **Warning**: Using the WPCS native whitelist comments is deprecated. Please use the PHPCS native &quot;phpcs:ignore Standard.Category.SniffName.ErrorCode&quot; annotations instead. Found: // WPCS: CSRF OK.\n (*WordPress.Security.NonceVerification.DeprecatedWhitelistCommentFound*).